chunks/chunks.go

```diff
@@ -352,30 +352,31 @@ func (s *Reader) Size() int64 {
 	return s.size
 }
 
+// Chunk returns a chunk from a given reference.
 func (s *Reader) Chunk(ref uint64) (chunkenc.Chunk, error) {
 	var (
-		seq = int(ref >> 32)
-		off = int((ref << 32) >> 32)
+		segmentSeq    = int(ref >> 32)
+		segmentOffset = int((ref << 32) >> 32)
 	)
-	if seq >= len(s.bs) {
-		return nil, errors.Errorf("reference sequence %d out of range", seq)
+	if segmentSeq >= len(s.bs) {
+		return nil, errors.Errorf("reference sequence %d out of range", segmentSeq)
 	}
-	b := s.bs[seq]
+	chunksBytes := s.bs[segmentSeq]
 
-	if off >= b.Len() {
-		return nil, errors.Errorf("offset %d beyond data size %d", off, b.Len())
+	if segmentOffset >= chunksBytes.Len() {
+		return nil, errors.Errorf("offset %d beyond data size %d", segmentOffset, chunksBytes.Len())
 	}
 	// With the minimum chunk length this should never cause us reading
 	// over the end of the slice.
-	r := b.Range(off, off+binary.MaxVarintLen32)
+	chunkBytes := chunksBytes.Range(segmentOffset, segmentOffset+binary.MaxVarintLen32)
```

Comment:
For a moment I was confused about this, different between `chunkBytes` and `chunksBytes` is very subtle. Can one of them be changed?